source/elements/oneMKL/source/architecture/other_architecture.inc.rst

```diff
@@ -11,10 +11,40 @@ This section covers all other features in the design of oneMKL architecture.
 
 .. _onemkl_spec_current_version:
 
-Current Version of this oneMKL Specification
+Specification Version and Compliance
 +++++++++++++++++++++++++++++++++++++++++++++
 
-This is the oneMKL specification which is part of the oneAPI specification version 1.0.0.
+Each oneMKL domain must define a preprocessor macro to represent the version of the specification that the implementation is compliant with.
+
+The macros for each domain are listed as follows:
+
+  | ONEMKL_BLAS_SPEC_VERSION
+  | ONEMKL_LAPACK_SPEC_VERSION
+  | ONEMKL_SPARSE_SPEC_VERSION
+  | ONEMKL_DFT_SPEC_VERSION
+  | ONEMKL_RNG_SPEC_VERSION
```

Comment:
I can see that Stats domain is missed: https://oneapi-spec.uxlfoundation.org/specifications/oneapi/v1.3-rev-1/elements/onemkl/source/domains/stats/onemkl_stats_overview

Reply:
Good catch. I could not find a corresponding namespace.
https://oneapi-spec.uxlfoundation.org/specifications/oneapi/v1.3-rev-1/elements/onemkl/source/architecture/architecture
Is this expected?

Reply:
We should discuss this in https://github.com/uxlfoundation/oneAPI-spec/issues/581 as well.

Reply:
Suggestion to add it as "STATS" - such namespace is used for statistics routines

Reply:
done

Reply:
Thanks!